File: back-end/powerfit/src/main/java/com/nocountry/powerfit/model/response/AuthResponse.java
package com.nocountry.powerfit.model.response;

import lombok.*;

import java.util.Date;

@Getter @Setter
@AllArgsConstructor
@NoArgsConstructor
@Builder
public class AuthResponse {
    private String token;
    private String message;
    private Long cartId;
    private String name;
}

File: front-end/src/app/model/auth-response.ts
export class AuthResponse {

  name:string;
  lastName:string;
  email:string;
  role:string;
  token:string;

  constructor(name:string, lastName:string, email:string, role:string, token:string){
    this.name = name;
    this.lastName = lastName;
    this.email = email;
    this.role = role;
    this.token = token;
  }
}
